qa/rpc-tests/sc_cert_listsinceblock.py

```diff
@@ -0,0 +1,381 @@
+        assert_true(found_bwt)
+        assert_true(found_out)
+
+        # calling the cmd targeting the block where the second certificate has been mined
+        mark_logs("Calling listsinceblock on Node2 for h={}, hash={}".format(block_heights_d[1], blocks_d[1]), self.nodes, DEBUG_MODE)
+        ret = self.nodes[2].listsinceblock(blocks_d[1], 1, False, True)
+        #pprint.pprint(ret)
+
+        cert = certs_d[0]
+        # first cert is there, it is mature and it refers to the block where it matured
+        # we should not see the standard output anymore 
+        mat_block_hash    = self.nodes[2].getblockhash(mat_height_d[0])
+        found = False
+        for x in ret['transactions']:
+            if x['txid'] == cert:
+                found = True
+                assert_equal(x['amount'],   Decimal('1.02'))
+                assert_equal(x['category'], 'receive')
+                assert_equal(x['blockhash'], mat_block_hash)
+                assert_equal(x['isBackwardTransfer'], True)
+        assert_true(found)
+
+        # cert 2 is not there (since-block is not part of the range) 
+        cert = certs_d[1]
+        found = False
+        for x in ret['transactions']:
+            if x['txid'] == cert:
+                found = True
+        assert_false(found)
+
+        # last cert is there and is immature, we have also standard output
+        cert = certs_d[2]
+        found_bwt = False
+        found_out = False
+        for x in ret['transactions']:
+            if x['txid'] == cert:
+                assert_equal(x['blockhash'], blocks_d[2])
+                if 'isBackwardTransfer' in x:
+                    assert_equal(x['category'], 'immature')
+                    assert_equal(x['amount'],   Decimal('3.02'))
+                    assert_equal(x['isBackwardTransfer'], True)
+                    found_bwt = True
+                else:
+                    assert_equal(x['category'], 'receive')
+                    assert_equal(x['amount'],   Decimal('0.003'))
+                    found_out = True
+        assert_true(found_bwt)
+        assert_true(found_out)
+
+        # calling the cmd targeting the block where the last certificate has been mined
+        # There must be no certificates at all, since none of them is contained or matures in this block range 
+        mark_logs("Calling listsinceblock on Node2 for h={}, hash={}".format(block_heights_d[2], blocks_d[2]), self.nodes, DEBUG_MODE)
+        ret = self.nodes[2].listsinceblock(blocks_d[2], 1, False, True)
+        #pprint.pprint(ret)
+        assert_true(len(ret['transactions']) == 0)
+
+        # reach the height of the second certificate and re-issue the command
+        c = self.nodes[0].getblockcount()
+        mark_logs("Node 0 generates {} block".format(mat_height_d[1] - c),self.nodes,DEBUG_MODE)
+        self.nodes[0].generate(mat_height_d[1] - c)
+        self.sync_all()
+        print "chain height = {}".format(self.nodes[0].getblockcount())
```

Comment:
Use the brackets for `print` for future compatibility with Python 3.

```suggestion
        print("chain height = {}".format(self.nodes[0].getblockcount()))
```

Reply:
Done